runtime/vm/jvminit.c

```diff
@@ -3801,6 +3801,9 @@ processVMArgsFromFirstToLast(J9JavaVM * vm)
 			if (requestedHeapSize <= maxHeapForCR) {
 				vm->extendedRuntimeFlags2 |= J9_EXTENDED_RUNTIME2_COMPRESS_OBJECT_REFERENCES;
 			}
+		} else if (compressed > nocompressed) {
+			/* use compressed refs if explicitly requested */
+			vm->extendedRuntimeFlags2 |= J9_EXTENDED_RUNTIME2_COMPRESS_OBJECT_REFERENCES;
```

Comment:
Suggestion: put small branches first:
```
		if (compressed > nocompressed) {
			/* use compressed refs if explicitly requested */
			vm->extendedRuntimeFlags2 |= J9_EXTENDED_RUNTIME2_COMPRESS_OBJECT_REFERENCES;
		} else if (compressed == nocompressed) {
			/* compressed refs by default */
...
```
This makes the code easier to follow because the conditions are near to each other.

Reply:
updated